src/main/scala/org/broadinstitute/hail/driver/VEP.scala

```diff
@@ -293,18 +302,26 @@ object VEP extends Command {
             printElement,
             _ => ())
             .map { s =>
-              val a = JSONAnnotationImpex.importAnnotation(parse(s), vepSignature)
-              val v = variantFromInput(inputQuery(a).get.asInstanceOf[String])
-              (v, a)
+              if(csq) {
+                csq_regex.findFirstIn(s).map(x => x.substring(4)) match {
```

Comment:
use 

```.findFirstIn(s).map(x => (VariantFromInput(s), x.substring(4))
  }
}.flatten```
 to keep things monadic, instead of producing nulls and filtering them out